### ISSUE TITLE
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Expected value at line 1 column 1 path $

### ISSUE THREAD

**Issue by zedwick** [QuiltServerTools/Ledger#46]
**Describe the bug**
Written book data sent to a client can disconnect the player with an error. The book can be in a chest, a dropped item or in the player's inventory.

```
[20:02:41] [Netty Epoll Server IO #1/ERROR]: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Expected value at line 1 column 1 path $
[20:02:41] [Server thread/INFO]: Zedwick lost connection: Internal Exception: io.netty.handler.codec.EncoderException: com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Expected value at line 1 column 1 path $                
[20:02:41] [Server thread/INFO]: Zedwick left the game
```

**To Reproduce**
1. Install Ledger
2. Get a blank written book
3. Enter a single '=' into the book
4. Click Done
5. Be immediately disconnected.

**Expected behavior**
Written books with an '=' should not disconnect the player.

**Device Info(please complete the following information):**
 - OS: Debian
 - Minecraft Version: 1.17
 - Version: 1.0.0

**Additional context**
This has been tested with and without Ledger; Without ledger this error does not occur, but with Ledger it does. I left Fabric Kotlin installed in all tests, and made no other changes.

I also had Fabric API, and FabricProxy mods installed.

**Comment by yitzy299** [comment]
Update to 1.0.1

**Comment by zedwick** [comment]
Ah, sorry. I didn't realise there was a new release available.

I can confirm this is fixed in 1.0.1. Thank you!
